examples/ding-dong-bot.ts

```diff
@@ -76,6 +76,15 @@ const bot = WechatyBuilder.build({
   // }
 })
 
+// 使用wechaty-puppet-wechat，需要设置uos: true
+// const bot = WechatyBuilder.build({
+//   name: 'WechatEveryDay',
```

Comment:
Please always use `ding-dong-bot` as the name of our getting started bot.

Reply:
done